### DISCUSSION TITLE
Can't receive SMS from some numbers

### DISCUSSION THREAD

**Discussion by threedflow** [Divested-Mobile/DivestOS-Build#164] [Q&A]
Hey,
so I have been using DivestOS for like a month and I really enjoy the privacy and security, but I experienced a problem that I think is major.

I can't receive SMS from some numbers. If someone wants to text me, the messages can't be delivered to me, but the funny part is some numbers CAN message me and the messages come. I have already checked APN points, used a different SMS/MMS application, searched through settings but still no success. I use avicii (DOS 17.1) and my career is based on Vodafone network. I use two SIMs. One for calls/SMS (calls are working flawlessly) and the second one for mobile data (working as well).

**Comment by SkewedZeppelin** [comment]
Is VoLTE working on your device? Like is there an HD symbol when you make a phone call?

**Reply by threedflow** [reply]
I have it enabled and I think it is working, but I turned the VoLTE icon off in the status bar as it was taking a lot of space there.

**Reply by SkewedZeppelin** [reply]
There is an HD icon that appears in the center of the screen below the call duration, do you see that one?

https://forum.f-droid.org/uploads/default/original/2X/5/53ee160037c0d5683a65629320d04f41c19f31bb.jpeg

**Reply by threedflow** [reply]
I found an American test number with a HD voice support and unfortunately, the HD icon didn't appear.

**Reply by threedflow** [reply]
So I guess VoLTE is not working either right?

**Reply by SkewedZeppelin** [reply]
SMS is handled over the VoLTE stack, so the issue here is likely that you're losing signal and having dropped messages (and likely calls too).

If you open the dialer and type in (but don't hit dial): `*#*#4636#*#*`, then go to phone info, 3dot in the top right, and `IMS service status`, does it show as registered?

**Reply by threedflow** [reply]
Nope
